corehq/apps/reports/standard/project_health.py

```diff
@@ -327,6 +329,49 @@ def previous_six_months(self):
             last_month_summary = this_month_summary
         return six_month_summary
 
+    def export_summary(self, six_months):
+        table = []
+        table.append('Six Month Performance Summary')
+        worksheet_headers = ['month', 'num_high_performing_users', 'num_low_performing_users',
+                             'total_active', 'total_inactive', 'total_num_users']
+        table.append([worksheet_headers])
+        for month_summary in six_months:
+            table[1].append([month_summary.month.isoformat(), month_summary.number_of_performing_users,
+                             month_summary.number_of_low_performing_users, month_summary.active,
+                             month_summary.inactive, month_summary.total_users_by_month])
+        return table
```

Comment:
This pattern is also repeated below, if it looks right, you could also define a `make_worksheet` function so you can make it really obvious what's what:
```python
return make_worksheet(
    title="Six Month Performance Summary",
    headers=[...],
    rows=[[monthly_summary.isoformat()...]
          for monthly_summary in six_months]
)
```
I'm not certain the pattern is identical in both cases, so it might not be worth it - just a thought.